test-suite/misc/deps-order-from.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Check that both coqdep and coqtop/coqc support -R
+# Check that both coqdep and coqtop/coqc take the latter -R
+# See bugs #11631, #14539
+rm -f misc/deps/test-from/A/C.vo misc/deps/test-from/B/C.vo misc/deps/test-from/D.vo misc/deps/test-from/E.vo
+tmpoutput=$(mktemp /tmp/coqcheck.XXXXXX)
+$coqdep -R misc/deps/test-from T misc/deps/test-from/D.v misc/deps/test-from/E.v 2>&1 > "$tmpoutput"
+diff -u --strip-trailing-cr misc/deps/deps-from.out "$tmpoutput" 2>&1
+R=$?
+times
+$coqc -R misc/deps/test-from T misc/deps/test-from/A/C.v 2>&1
+$coqc -R misc/deps/test-from T misc/deps/test-from/B/C.v 2>&1
+$coqc -R misc/deps/test-from T misc/deps/test-from/D.v 2>&1
+$coqc -R misc/deps/test-from T misc/deps/test-from/E.v 2>&1
```

Comment:
Why all the `2>&1`?

Reply:
If there are warnings, we may want to know about them, no?